pkg/image/export/main.go

```diff
@@ -267,35 +267,44 @@ func getK3sUpgradeImages(rancherVersion string, k3sData map[string]interface{})
 			continue
 		}
 
-		if rancherVersion != "dev" {
-			maxVersion, _ := releaseMap["maxChannelServerVersion"].(string)
-			maxVersion = strings.TrimPrefix(maxVersion, "v")
-			if maxVersion == "" {
-				continue
-			}
-			minVersion, _ := releaseMap["minChannelServerVersion"].(string)
-			minVersion = strings.Trim(minVersion, "v")
-			if minVersion == "" {
-				continue
-			}
+		maxVersion, _ := releaseMap["maxChannelServerVersion"].(string)
```

Comment:
Is there value in at least logging the error if we encounter it?

Reply:
This is usually ran in a CI process and since it doesn't cause a build failure it likely won't be noticed but can't hurt.

Reply:
@nickgerace actually what I said applies to the actual errors but this is just an ok check to keep it safe.

Reply:
Sounds goo d to me. Thanks